### PR TITLE
Fireworks add gem rpz view fix

### PR DESCRIPTION
**Pull request by jshlee**

### CODE REVIEW

File: Geometry/GEMGeometry/test/GEMGeometryAnalyzer.cc [hunk 1]
```diff
@@ -43,20 +45,20 @@ class GEMGeometryAnalyzer : public edm::EDAnalyzer {
   const std::string myName_;
   std::ofstream ofos;
 };
-
+using namespace std;
 GEMGeometryAnalyzer::GEMGeometryAnalyzer( const edm::ParameterSet& /*iConfig*/ )
```
Comment (dildick) [inline Geometry/GEMGeometry/test/GEMGeometryAnalyzer.cc:49]: Official CMSSW does not allow the use of "using namespace std;"

File: Fireworks/Muons/plugins/FWGEMRecHitProxyBuilder.cc [hunk 2]
```diff
@@ -19,6 +19,7 @@
 #include "Fireworks/Core/interface/fwLog.h"
 
 #include "DataFormats/GEMRecHit/interface/GEMRecHitCollection.h"
+using namespace std;
```
Comment (dildick) [inline Fireworks/Muons/plugins/FWGEMRecHitProxyBuilder.cc:23]: Official CMSSW does not allow the use of "using namespace std;"

File: Fireworks/Muons/plugins/FWGEMDigiProxyBuilder.cc [hunk 3]
```diff
@@ -23,6 +23,10 @@
 
 #include "DataFormats/GEMDigi/interface/GEMDigiCollection.h"
 #include "DataFormats/GEMDigi/interface/GEMCSCPadDigiCollection.h"
+#include "Geometry/GEMGeometry/interface/GEMGeometry.h"
+#include "Geometry/GEMGeometry/interface/GEMEtaPartition.h"
+
+using namespace std;
```
Comment (dildick) [inline Fireworks/Muons/plugins/FWGEMDigiProxyBuilder.cc:30]: Official CMSSW does not allow the use of "using namespace std;"

File: Fireworks/Muons/plugins/FWGEMRecHitProxyBuilder.cc (hunk 2 above)
Comment (jshlee) [inline Fireworks/Muons/plugins/FWGEMRecHitProxyBuilder.cc:23]: I didn't know.
I was just using them for testing. I can remove them now.

Comment (slava77) [inline Fireworks/Muons/plugins/FWGEMRecHitProxyBuilder.cc:23]: On 11/14/13, 9:38 AM, Sven Dildick wrote:

> In Fireworks/Muons/plugins/FWGEMRecHitProxyBuilder.cc:
> 
> > @@ -19,6 +19,7 @@
> >  #include "Fireworks/Core/interface/fwLog.h"
> > 
> >  #include "DataFormats/GEMRecHit/interface/GEMRecHitCollection.h"
> > +using namespace std;
> 
> Official CMSSW does not allow the use of "using namespace std;"

This applies only to header files.
(I personally prefer the namespaces mentioned explicitly)

```
    --slava
```

> —
> Reply to this email directly or view it on GitHub
> https://github.com/gem-sw/cmssw/pull/31/files#r7652771.

## 

---

 Vyacheslav (Slava) Krutelyov
 TAMU: Physics Dept Texas A&M MS4242, College Station, TX 77843-4242
 CERN: 42-R-027
 AIM/Skype: siava16              googleTalk: slava77@gmail.com
 (630) 291-5128  Cell (US)       +41 76 275 7116 Cell (CERN)

---

**Pull request merged**